src/bot.py

```diff
@@ -34,10 +39,53 @@
 maxinterval = config['maxinterval']
 
 cooldown = randint(mininterval, maxinterval) * 24 * 60 * 60
+

```

Comment:
Why did you add a newline above the comment which explained the purpose of the `cooldown` variable?